misc/python/materialize/output_consistency/validation/error_message_normalizer.py

```diff
@@ -0,0 +1,24 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import re
+
+
+class ErrorMessageNormalizer:
+    def normalize(self, error_message: str) -> str:
+        # replace source prefix in column
+        normalized_message = re.sub(
+            'column "[^.]*\\.', 'column "<source>.', error_message
+        )
+        normalized_message = normalized_message.replace("Evaluation error: ", "")
+
+        if normalized_message.startswith("mz_timestamp out of range ("):
```

Comment:
It is because of this deviation. Let me know in case this should deserve a ticket:
```
--   Value 1 (Dataflow rendering): 'mz_timestamp out of range' (type: <class 'str'>)
--   Value 2 (Constant folding): 'mz_timestamp out of range (Integer, numeric, and text casts to mz_timestamp must be in the form of whole milliseconds since the Unix epoch. Values with fractional parts cannot be converted to mz_timestamp.)' (type: <class 'str'>)
```

Reply:
Kind of strange that one has an explanation and the other doesn't. You could use `git blame` to find the person who added the explanation and ask them if they forgot/plan to add it in the other case too.

Reply:
I approached Cara and am awaiting her reply.